packages/amplify-codegen/src/commands/models.js

```diff
@@ -86,7 +86,7 @@ async function generateModels(context) {
 
   const generateIndexRules = readFeatureFlag('codegen.generateIndexRules');
   const emitAuthProvider = readFeatureFlag('codegen.emitAuthProvider');
-  const usePipelinedTransformer = readFeatureFlag('graphQLTransformer.useExperimentalPipelinedTransformer')
+  const usePipelinedTransformer = readFeatureFlag('graphQLTransformer.useExperimentalPipelinedTransformer');
```

Comment:
seems like some of these files got reformatted unrelated to my change, lmk if prefer to not have in this PR

Reply:
It is better not to include the lint change since it is too many or you can comment on the non-lint change so that I can locate it easily

Reply:
ok will remove, thanks